### PR TITLE
Unattended changes the login screen in kibana [unified]

### LINKED ISSUE #943: Add custom login page in unattended script instead in the OVA/AMI build process
Hello team,

Both the OVA and AWS AMI are built using the unattended script, in each of them, a configuration is made to have a personalized Kibana login page that changes the background, the title of the window and the title that It is displayed above the username and password forms.

It would be convenient to study and implements those procedures in the unattended script instead of performing them during the installation of the OVA/AMI.

Current:
image

Desired:
image

Regards, Raúl.

### PR DESCRIPTION
**Pull request by verdx**
|Related issue|
|---|
| closes #943 |

## Description

Before starting the kibana service we use the same code used previously in the ova to change the kibana interface parameters and files. When kibana is started, the wazuh logo is shown, instead of the opendistro one.

## Tests
Login screen after all-in-one installation in centos 7:
custom_login_centos7

The all-in-one installation runs correctly in Centos 7 and debian 9

### CODE REVIEW

**Review by lastranikos — APPROVED** [review]
LGTM

File: unattended_scripts/install_functions/opendistro/kibana.sh [hunk 1]
```diff
@@ -87,6 +97,16 @@ configureKibana() {
...
 
+    # Edit window title
+    eval "sed -i 's/null, \"Elastic\"/null, \"Wazuh\"/g' /usr/share/kibana/src/core/server/rendering/views/template.js ${debug}"
+    eval "curl -so ~/custom_welcome.tar.gz https://wazuh-demo.s3-us-west-1.amazonaws.com/custom_welcome_opendistro_docker.tar.gz ${debug}"
+    eval "tar -xf ~/custom_welcome.tar.gz -C ~ ${debug}"
+    eval "cp ~/custom_welcome/wazuh_logo_circle.svg /usr/share/kibana/src/core/server/core_app/assets/ ${debug}"
+    eval "cp ~/custom_welcome/wazuh_wazuh_bg.svg /usr/share/kibana/src/core/server/core_app/assets/ ${debug}"
+    eval "cp -f ~/custom_welcome/template.js.hbs /usr/share/kibana/src/legacy/ui/ui_render/bootstrap/template.js.hbs ${debug}"
+    eval "curl -so ~/customWelcomeKibana.css ${resources}/open-distro/kibana/customWelcomeKibana.css ${debug}"
+    eval "cat ~/customWelcomeKibana.css | tee -a /usr/share/kibana/src/core/server/core_app/assets/legacy_light_theme.css ${debug}"
```
Comment (rauldpm) [inline unattended_scripts/install_functions/opendistro/kibana.sh:108]: Maybe we should create a function, e.g. modifyKibanaLogin() or similar with this code, and call it from configureKibana() and configureKibanaAIO(). 

Also, add a empty line between 101 and 102, as these lines references different things, the line 101 modifies the title of the browser page and lines 102 onwards modify the background and logos, the comment on line 100 can imply that the entire block does that when it is not.

**Review by rauldpm — CHANGES_REQUESTED** [review]
GJ, just a minor change

File: unattended_scripts/install_functions/opendistro/kibana.sh (hunk 1 above)
Comment (verdx) [reply unattended_scripts/install_functions/opendistro/kibana.sh]: Fixed in 2410e85817123d565e41a46c548b0b1f959c461f

**Review by alberpilot — APPROVED** [review]
LGTM

**Review by rauldpm — APPROVED** [review]
LGTM

**Pull request merged**